src/ProjectFeedbackSchema.js

```diff
@@ -3,45 +3,39 @@
 // la _rúbrica_ y que las llaves de `reviewerSurveyResults` hacen
 // referencia a preguntas válidas del `ReviewerSurvey` en cuestión.
 
-
 module.exports = (conn) => {
   const ProjectFeedbackSchema = new conn.Schema({
-    project: {
+    cohortProject: {
+      type: conn.Schema.Types.ObjectId,
+      ref: 'CohortProject',
+      required: true,
+    },
+    cohortMembership: {
       type: conn.Schema.Types.ObjectId,
-      ref: 'Project',
+      ref: 'CohortMembership',
       required: true,
     },
-    cohort: {
+    createdBy: {
       type: conn.Schema.Types.ObjectId,
-      ref: 'Cohort',
+      ref: 'User',
       required: true,
     },
-    // `uid` debería ser un `ObjectId` que apunte al `user`????
-    uid: { type: String, required: true }, // Firebase UID
-    // `uid` de la evaluadora...
-    createdBy: { type: String, required: true }, // Firebase UID
-    createdAt: { type: Date, required: true, default: Date.now },
-    // `rubric` hace referencia a la versión (major) de la rúbrica
+    createdAt: {
+      type: Date,
+      default: Date.now,
+    },
     rubric: {
       type: String,
       required: true,
-      enum: ['1', '2'],
     },
-    // `rubricResults` debería ser requerido???
     rubricResults: {
       type: Map,
       of: Number,
-      required: true,
     },
-    // `reviewerSurvey` es el id de un objeto de tipo ReviewerSurvey
-    reviewerSurvey: {
-      type: conn.Schema.Types.ObjectId,
-      ref: 'ReviewerSurvey',
-      required: true,
+    reviewerSurveyResults: {
```

Comment:
Al igual que con las _sub-colecciones_ de los perfiles académicos, me pregunto si queremos/necesitamos guardar esta subcolección directamente sobre el documento de project feedback. Viendo el schema de `ReviewAnswerSchema` me confunde ver ahí un nombre de colección, con lo cual asumí que estos docs se almacenarían por separado y no directamente sobre project feedback... :thinking:

Reply:
Se realizo el cambio :)
**Model One-to-Many Relationships with Embedded Documents**